transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java

```diff
@@ -67,6 +68,22 @@
      */
     private Map<EventExecutorGroup, ChannelHandlerInvoker> childInvokers;
 
+    /**
+     * This is the head of a linkedlist that is processed by {@link #callHandlerAddedForAllHandlers()} and so process
+     * all the pending {@link #callHandlerAdded0(AbstractChannelHandlerContext)}.
+     *
+     * We only keep the head because it is expected that the size of the list we be small, and used infrequently.
```

Comment:
the size of the list we be small, and used infrequently. -> the list is used infrequently and its size is small.